app/src/main/kotlin/org/fossify/keyboard/services/SimpleKeyboardIME.kt

```diff
@@ -465,6 +465,7 @@ class SimpleKeyboardIME : InputMethodService(), OnKeyboardActionListener, Shared
             LANGUAGE_HEBREW -> R.xml.keys_letters_hebrew
             LANGUAGE_ITALIAN -> R.xml.keys_letters_italian
             LANGUAGE_KABYLE_AZERTY -> R.xml.keys_letters_kabyle_azerty
+            LANGUAGE_LITHUANIAN -> R.xml.keys_letters_latvian
```

Comment:
Another typo. It should be `LANGUAGE_LATVIAN`.

```suggestion
            LANGUAGE_LATVIAN -> R.xml.keys_letters_latvian
```